client/src/components/Sidebar/Sidebar.tsx

```diff
@@ -180,8 +184,9 @@ export const Sidebar: React.FunctionComponent<SidebarProps> = (
       <aside
```

Comment:
Question: would `<nav>` be more appropriate than `<aside>` here?

Reply:
`<aside>` is what we use for the existing sidebar. But perhaps we should be using `<nav>` for the main menu portion of it?

Reply:
That feels right to me